### ISSUE TITLE
Update Slashing (may need versioning)

### ISSUE THREAD

**Issue by joepetrowski** [w3f/polkadot-wiki-old#253]
https://wiki.polkadot.network/en/latest/polkadot/learn/staking/#slashing

Offline slash grace and unstake threshold are no longer in Polkadot master or Kusama. We need to update this section.

The page is a mix of things that are true for Alexander and true for Kusama, but not on both. We should decide if our documentation supports Alexander at all via versioning or if we just focus on Kusama/master. I vote the latter.

**Comment by lsaether** [comment]
When possible, we can describe mechanisms as generic over chains. When necessary, we cite parameters as specific to the chain. We should track polkadot master. Not opposed to versioning at some point but would need to lay out the groundwork to do this properly, not worth it for Alexander imo which anyways is now becoming obsolete.

**Comment by joepetrowski** [comment]
Yeah I agree, let's keep it up to date with master and not worry about Alex.

**Comment by lsaether** [comment]
@ansonla3 

We should have a clear description of the currently **three** slashable offences, how much you get slashed for committing them and why a validator would do them (and how to protect against it).

- Unresponsiveness.
- Babe Equivocation.
- Grandpa Equivocation
